src/routes/download/all-scheme-payment-data.js

```diff
@@ -0,0 +1,19 @@
+import Wreck from '@hapi/wreck'
+import { buildBackendUrl } from '../../api/build-backend-url.js'
+
+export const downloadAllSchemePaymentData = {
+  method: 'GET',
+  path: '/all-scheme-payment-data/file',
+  handler: async function (_request, h) {
+    const backendUrl = buildBackendUrl('/file')
+    const stream = await Wreck.request('get', backendUrl)
+
+    return h
+      .response(stream)
+      .type('application/csv')
```

Comment:
'application/csv' isn't a valid type.

Should be `text/csv`

https://www.iana.org/assignments/media-types/media-types.xhtml